tests/group/group_common.h

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+//
+//  SYCL 1.2.1 Conformance Test Suite
+//
+//  Copyright:	(c) 2018 by Codeplay Software LTD. All Rights Reserved.
+//
+// Provides common methods for group tests
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_GROUP_GROUP_COMMON_H
+#define __SYCLCTS_TESTS_GROUP_GROUP_COMMON_H
+
+#include "../common/common.h"
+#include <array>
+
+namespace {
+
+/** Retrieve group objects and store them
+ */
+template <typename kernelT, int numDims, size_t numItems>
+void store_group_instances(
+    std::array<cl::sycl::group<numDims>, numItems>& items)
```

Comment:
I am unsure it works since `cl::sycl::group<numDims>` does not have a public default constructor.